ui/pages/Search/components/VariantSearchResults.jsx

```diff
@@ -91,6 +94,24 @@ const BaseVariantSearchResults = ({
           <Variants variants={searchedVariants} />
         </Grid.Column>
       </Grid.Row>
+      {searchHash &&
+        <LargeRow>
+          <Grid.Column width={11} floated="right" textAlign="right">
+            <ReduxFormWrapper
```

Comment:
presumably, this is either the same or very similar to the pagination component at the top of the page. therefore, you should  make it its own component and then use that component here and at the top of the page